bindings/opencv/maskrcnn/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(aditof-opencv-maskrcnn)
+
+add_executable(${PROJECT_NAME} main.cpp)
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/frozen_inference_graph.pb
```

Comment:
Can't we just give the user some documentation on how to get this files? They seem a bit big to be included in the repo and could prolong the duration of `cloning` the repo even when this example is not needed. (frozen_inference_graph.pb is 64MB)

Reply:
If the files are binary and/or have large sizes, then we shouldn't include them in the repository.

Reply:
solved.